tests/integration/new_relations/test_new_relations_1.py

```diff
@@ -526,12 +514,12 @@ async def test_admin_role(ops_test: OpsTest):
                 )
                 assert version == data
 
-                # Write some data (it should fail in the "postgres" database).
+                # Write some data (it should fail in the default database name).
                 random_name = (
                     f"test_{''.join(secrets.choice(string.ascii_lowercase) for _ in range(10))}"
                 )
-                should_fail = database == "postgres"
-                cursor.execute(f"CREATE TABLE {random_name}(data TEXT);")
+                should_fail = database == DATABASE_DEFAULT_NAME
+                cursor.execute(f"CREATE SCHEMA test; CREATE TABLE test.{random_name}(data TEXT);")
```

Comment:
PG 16 doesn't give access to the public schema by default. See https://github.com/canonical/postgresql-k8s-operator/pull/788#discussion_r1954998265